app/src/main/java/ar/com/wolox/android/example/BootstrapApplication.kt

```diff
@@ -0,0 +1,54 @@
+package ar.com.wolox.android.example
+
+import ar.com.wolox.android.BuildConfig
+import ar.com.wolox.android.example.di.DaggerAppComponent
+import ar.com.wolox.wolmo.core.WolmoApplication
+import ar.com.wolox.wolmo.networking.di.DaggerNetworkingComponent
+import ar.com.wolox.wolmo.networking.di.NetworkingComponent
+import com.google.gson.FieldNamingPolicy
+import com.readystatesoftware.chuck.ChuckInterceptor
+import com.squareup.leakcanary.LeakCanary
+import dagger.android.AndroidInjector
+import okhttp3.logging.HttpLoggingInterceptor
+import okhttp3.logging.HttpLoggingInterceptor.Level
+
+class BootstrapApplication : WolmoApplication() {
+
+    override fun onInit() {
```

Comment:
I would add a line inside the method saying something like "Initialize Application wide stuff here".